modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/TSLsAppris_conf.pm

```diff
@@ -194,150 +91,79 @@ sub default_options {
-      },
-      -flow_into => {
-          '2' => ['download_appris'],
-      },
+    {  -logic_name => 'APPRIS_TSL_Datachecks',
+       -module     => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
+       -flow_into  => {
+		                 '1->A' => ['APPRIS_TSL_Critical_Datachecks'],
+		                 'A->1' => ['APPRIS_TSL_Advisory_Datachecks'],		                       
+                       },          
     },
-
     {
-      -logic_name => 'download_appris',
-      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
-      -rc_name => 'default',
-      -parameters => {
-          cmd => 'wget -P'.catfile($self->o('working_dir'), 'appris_'.$self->o('ensembl_release')).' -qq "'.$self->o('appris_ftp_base').'/#species_alias#.#cs_name#.e'.$self->o('ensembl_release').'.appris_data.principal.txt"',
-          return_codes_2_branches => {'8' => 2},
-      },
-      -flow_into => {
-          '1' => ['load_appris'],
-      },
+      -logic_name      => 'APPRIS_TSL_Critical_Datachecks',
+      -module          => 'Bio::EnsEMBL::DataCheck::Pipeline::RunDataChecks',
+      -parameters      => {
+                            datacheck_names => ['AttribValuesExist'],
+                            history_file    => $self->o('history_file'),
+                            failures_fatal  => 1,
+                          },
+      -max_retry_count => 1,
+      -hive_capacity   => 50,
+      -batch_size      => 10,
+      -rc_name         => 'normal',
     },
-
     {
-      -logic_name => 'load_appris',
-      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
-      -rc_name => 'default',
-      -parameters => {
-          cmd => 'perl '.$self->o('load_appris_script').
-            ' -h #expr(#target_db#->{-host})expr#'.
-            ' -D #expr(#target_db#->{-dbname})expr#'.
-            ' -P #expr(#target_db#->{-port})expr#'.
-            ' -u #expr(#target_db#->{-user})expr#'.
-            ' -p #expr(#target_db#->{-pass})expr#'.
-            ' -cs_version #cs_name#'.
-            ' -write -verbose'.
-            ' -infile '.catfile($self->o('working_dir'), 'appris_'.$self->o('ensembl_release'), '#species_alias#.#cs_name#.e'.$self->o('ensembl_release').'.appris_data.principal.txt'),
-      },
-    },
-
-  );
-
-  foreach my $analysis (@analysis) {
-    $analysis->{-max_retry_count} = 1 unless (exists $analysis->{-max_retry_count});
-  }
-  return \@analysis;
+      -logic_name      => 'APPRIS_TSL_Advisory_Datachecks',
+      -module          => 'Bio::EnsEMBL::DataCheck::Pipeline::RunDataChecks',
+      -parameters      => {
+                            datacheck_names => ['AttribValuesCoverage'],
+                            history_file    => $self->o('history_file'),
+                            failures_fatal  => 0,
+                          },
+      -max_retry_count => 1,
+      -hive_capacity   => 50,
+      -batch_size      => 10,
+      -rc_name         => 'normal',
+    }
```

Comment:
I suggest flowing to an email notification module here, _a la_ https://github.com/Ensembl/ensembl-production/blob/release/99/modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/DumpCore_conf.pm#L547, otherwise you have to manually inspect the output to see if there was a failure.

Reply:
Make sense @james-monkeyshines